.github/workflows/wf-build-test.yml

```
@@ -33,59 +26,56 @@
+        $coverage = if ("${{ matrix.dotnet-framework.coverage }}" -eq "true") { "/p:CollectCoverage=true /p:CoverletOutputFormat=opencover" } else { "" }
+        $noBuild = if ("${{ matrix.dotnet-framework.no-build }}" -eq "true") { "--no-build" } else { "" }
+        dotnet test -c Release -f ${{ matrix.dotnet-framework.framework }} --nologo $noBuild $coverage
 
-    - name: Run test .NET 6.0
-      working-directory: QRCoderTests
-      run: dotnet test -c Release -f net6.0 --nologo --no-build /p:CollectCoverage=true /p:CoverletOutputFormat=opencover
+  additional-tests:
+    needs: build
+    runs-on: windows-2019
+    steps:
+    - name: Download artifacts
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)